src/commands/refund.rs

```diff
@@ -0,0 +1,50 @@
+use candid::Encode;
+use clap::Parser;
+use ic_sns_sale::pb::v1::ErrorRefundIcpRequest;
+
+use crate::{
+    lib::{
+        signing::{sign_ingress_with_request_status_query, IngressWithRequestId},
+        AnyhowResult, TargetCanister,
+    },
+    SnsCanisterIds,
+};
+
+use super::transfer;
+
+/// Signs a message to request a refund from the SNS swap canister.
+/// If the swap was aborted or failed, or some of your contributed ICP never made it into a neuron,
+/// this command can retrieve your unused ICP, minus transaction fees.
```

Comment:
Nit: Could you copy this to mod.rs as that is where clap pulls the descriptions displayed when using `sns-quill --help`

Reply:
It only does that if you provide one; if the full help message should be displayed in all circumstances then the same effect is achieved by simply leaving it blank, no?

Reply:
I didn't know that Clap was capable of doing that routing. Leaving it blank removes comment duplication so SGTM